libraries/AP_HAL_ChibiOS/hwdef/VUAV-V7pro/README.md

```diff
@@ -0,0 +1,607 @@
+   </tr>
+   <tr>
+   <td>4</td>
+   <td>GND</td>
+   <td>GND</td>
+   </tr>
+   </tbody>
+   </table>
+
+### RSSI
+
+   <table border="1" class="docutils">
+   <tbody>
+   <tr>
+   <th>Pin</th>
+   <th>Signal</th>
+   <th>Volt</th>
+   </tr>
+   <tr>
+   <td>1</td>
+   <td>VCC</td>
+   <td>+5V</td>
+   </tr>
+   <tr>
+   <td>2</td>
+   <td>RSSI</td>
+   <td>up to +3.3V</td>
+   </tr>
+   <tr>
+   <td>3</td>
+   <td>SBUS_OUT</td>
+   <td>+3.3</td>
+   </tr>
+   <tr>
+   <td>4</td>
+   <td>GND</td>
+   <td>GND</td>
+   </tr>
+   </tbody>
+   </table>
+
+### RS232
+
+   <table border="1" class="docutils">
+   <tbody>
+   <tr>
+   <th>Pin </th>
+   <th>Signal </th>
+   <th>Volt </th>
+   </tr>
+   <tr>
+   <td>1</td>
+   <td>VCC</td>
+   <td>+5V</td>
+   </tr>
+   <tr>
+   <td>2</td>
+   <td>TX (OUT)</td>
+   <td>+3.3V</td>
+   </tr>
+   <tr>
+   <td>3</td>
+   <td>RX (IN)</td>
+   <td>+3.3V</td>
+   </tr>
+   <tr>
+   <td>4</td>
+   <td>GND</td>
+   <td>GND</td>
+   </tr>
+   </tbody>
+   </table>
+
+### DEBUG
+
+   <table border="1" class="docutils">
+   <tbody>
+   <tr>
+   <th>Pin </th>
+   <th>Signal </th>
+   <th>Volt </th>
+   </tr>
+   <tr>
+   <td>1</td>
+   <td>VCC</td>
+   <td>+5V</td>
+   </tr>
+   <tr>
+   <td>2</td>
+   <td>TX (OUT)</td>
+   <td>+3.3V</td>
+   </tr>
+   <tr>
+   <td>3</td>
+   <td>RX (IN)</td>
+   <td>+3.3V</td>
+   </tr>
+   <tr>
+   <td>4</td>
+   <td>SWDIO</td>
+   <td>+3.3V</td>
+   </tr>
+   <tr>
+   <td>5</td>
+   <td>SWCLK</td>
+   <td>+3.3V</td>
+   </tr>
+   <tr>
+   <td>6</td>
+   <td>GND</td>
+   <td>GND</td>
+   </tr>
+   </tbody>
+   </table>
+
+### ADC
+
+   <table border="1" class="docutils">
+   <tbody>
+   <tr>
+   <th>Pin</th>
+   <th>Signal</th>
+   <th>Volt</th>
+   </tr>
+   <tr>
+   <td>1</td>
+   <td>VCC</td>
+   <td>+5V</td>
+   </tr>
+   <tr>
+   <td>2</td>
+   <td>ADC_3V3</td>
+   <td>up to +3.3V</td>
+   </tr>
+   <tr>
+   <td>3</td>
+   <td>ADC_6V6</td>
+   <td>up to +6.6V</td>
+   </tr>
+   <tr>
+   <td>4</td>
+   <td>GND</td>
+   <td>GND</td>
+   </tr>
+   </tbody>
+   </table>
+
+### POWER1
+
+   <table border="1" class="docutils">
+   <tbody>
+   <tr>
+   <th>Pin</th>
+   <th>Signal</th>
+   <th>Volt</th>
+   </tr>
+   <tr>
+   <td>1</td>
+   <td>VCC IN</td>
+   <td>+5V</td>
+   </tr>
+   <tr>
+   <td>2</td>
+   <td>VCC IN</td>
+   <td>+5V</td>
+   </tr>
+   <tr>
+   <td>3</td>
+   <td>CURRENT</td>
+   <td>up to +3.3V</td>
+   </tr>
+   <tr>
+   <td>4</td>
+   <td>VOLTAGE</td>
+   <td>up to +3.3V</td>
+   </tr>
+   <td>5</td>
+   <td>GND</td>
+   <td>GND</td>
+   </tr>
+   <td>6</td>
+   <td>GND</td>
+   <td>GND</td>
+   </tr>
+   </tbody>
+   </table>
+
+### POWER2
+
+   <table border="1" class="docutils">
+   <tbody>
+   <tr>
+   <th>Pin</th>
+   <th>Signal</th>
+   <th>Volt</th>
+   </tr>
+   <tr>
+   <td>1</td>
+   <td>VCC IN</td>
+   <td>+5V</td>
+   </tr>
+   <tr>
+   <td>2</td>
+   <td>VCC IN</td>
+   <td>+5V</td>
+   </tr>
+   <tr>
+   <td>3</td>
+   <td>CAN_H</td>

```

Comment:
which can bus is used?should be labelled

Reply:
The chip uses MCP2542FD-E/MF,Should it be reflected above?

Reply:
not necessary.....just users need to know what can bus number belongs to a given can bus pinout, because some can devices can not co-exist on the same bus

Reply:
Okay, I will mark whether it is CAN1 or CAN2

Reply:
still needs the pin label updated